analysis/model/make_model_input.R

```diff
@@ -94,450 +108,485 @@ for (i in 1:nrow(active_analyses)) {
-  
-  if (active_analyses$analysis[i]=="sub_male_age_60_79") {
-    
-    print('Make model input: sub_male_age_60_79')
-    
-    df <- input[input$sub_bin_covid19_confirmed_history==FALSE & 
-                  input$cov_cat_sex=="Male" & 
-                  input$cov_num_age>=60 &
-                  input$cov_num_age<80,]
-    
-    df[,c(colnames(df)[grepl("sub_",colnames(df))],"cov_cat_sex")] <- NULL
+
+  # Make model input: prior parkinson true -------------------------------------
+
+  if (active_analyses$analysis[i]=="sub_prior_parkinson_true") {
+
+    print('Make model input: sub_prior_parkinson_true')
+
+    df <- input[input$sub_bin_covid19_confirmed_history==FALSE &
+                  input$cov_bin_history_any_dementia == TRUE,]
+
+    df[,c(colnames(df)[grepl("sub_",colnames(df))],"cov_bin_history_any_dementia")] <- NULL 
     
+
     check_vitals(df)
     readr::write_rds(df, file.path("output", paste0("model_input-",active_analyses$name[i],".rds")), compress = "gz")
     print(paste0("Saved: output/model_input-",active_analyses$name[i],".rds"))
     rm(df)
-    
+
   }
-  
-  # Make model input: sub_male_age_80_110 ----------------------------------------
-  
-  if (active_analyses$analysis[i]=="sub_male_age_80_110") {
-    
-    print('Make model input: sub_male_age_80_110')
-    
-    df <- input[input$sub_bin_covid19_confirmed_history==FALSE & 
-                  input$cov_cat_sex=="Male" & 
-                  input$cov_num_age>=80 &
-                  input$cov_num_age<111,]
-    
-    df[,c(colnames(df)[grepl("sub_",colnames(df))],"cov_cat_sex")] <- NULL
+
+  # Make model input: prior parkison false -------------------------------------
+
+  if (active_analyses$analysis[i]=="sub_prior_parkinson_true") {
+
+    print('Make model input: sub_prior_parkinson_false')
+
+    df <- input[input$sub_bin_covid19_confirmed_history==FALSE &
+                  input$cov_bin_history_any_dementia == FALSE,]
+
+    df[,c(colnames(df)[grepl("sub_",colnames(df))],"cov_bin_history_any_dementia")] <- NULL 
     
     check_vitals(df)
     readr::write_rds(df, file.path("output", paste0("model_input-",active_analyses$name[i],".rds")), compress = "gz")
     print(paste0("Saved: output/model_input-",active_analyses$name[i],".rds"))
     rm(df)
-    
+
   }
-  
-  # Make model input: sub_female_age_18_39 ---------------------------------------
-  
-  if (active_analyses$analysis[i]=="sub_female_age_18_39") {
-    
-    print('Make model input: sub_female_age_18_39')
-    
-    df <- input[input$sub_bin_covid19_confirmed_history==FALSE & 
-                  input$cov_cat_sex=="Female" & 
-                  input$cov_num_age>=18 &
-                  input$cov_num_age<40,]
-    
-    df[,c(colnames(df)[grepl("sub_",colnames(df))],"cov_cat_sex")] <- NULL
+
+  # Make model input: prior prior parkinson risk true --------------------------
+
+  if (active_analyses$analysis[i]=="sub_prior_parkinson_risk_true") {
```

Comment:
This comment applies to all new subgroups: Please could you rename the subgroups in active analyses so they are consistent with the variables you are using - this makes it easier for people unfamiliar with the project to track what is going on. For example, the subgroup variable here is called "sub_prior_parkinson_risk_true" and it uses the variable "cov_bin_history_parkinson_risk". I suggest we use "sub_history_parkinson_risk_true" to match "cov_bin_history_parkinson_risk".

Reply:
Done.